content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+- :guilabel:`Demo`: files are created and accepted automatically in demo mode but are **not** sent
+  to the |DGI|. For this reason, rejection errors will not appear in this mode. Every internal
+  validation can be tested in demo mode. Avoid selecting this option in a production database.
+
+Then, enter the :guilabel:`Uruware Credentials`:
+
+- :guilabel:`Uruware WS Password`
+- :guilabel:`Commerce Code`
+- :guilabel:`Terminal Code`
+
+.. image:: uruguay/electronic-invoice-data.png
+   :alt: Required information for electronic invoice.
+   :align: center
+
+.. note::
+   These credentials can be obtained from the email received when the :ref:`Uruguay account was
+   created <l10n_uy/uruware-account>`.
+
+Configure master data
+---------------------
+
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+
+The :doc:`chart of accounts <../accounting/get_started/chart_of_accounts>` is installed by default
+as part of the set of data included in the localization module, the accounts are mapped
+automatically in taxes, default accounts payable, and default accounts receivable.
+
+Accounts can by added or deleted according to the company's needs.
+
+.. seealso::
+   :doc:`../accounting/get_started/chart_of_accounts`
+
+Contacts
+~~~~~~~~
+
+To create a contact, navigate to :menuselection:`Contacts app` and select :guilabel:`New`. Then
+enter the following information:
+
+- :guilabel:`Company Name`
+- :guilabel:`Address`:
+
+  - :guilabel:`Street`: required to confirm an electronic invoice.
+  - :guilabel:`City`
+  - :guilabel:`State`
+  - :guilabel:`ZIP`
+  - :guilabel:`Country`: required to confirm an electronic invoice.
+
+- :guilabel:`Identification Number`:
+
+  - :guilabel:`Type`: select a identification number type.
+  - :guilabel:`Number`: required to confirm an electronic invoice.
+
+Review taxes
+~~~~~~~~~~~~
+
+As part of the Uruguay localization module, taxes are automatically created with its configuration
+and related financial accounts.
+
+.. image:: uruguay/taxes.png
+   :align: center
+   :alt: Taxes for Uruguay.
+
+Review document types
+~~~~~~~~~~~~~~~~~~~~~
+
+Some accounting transactions like *Customer Invoices* and *Vendor Bills* are classified by document
+types. These are defined by the government fiscal authorities, in this case by the |DGI|.
+
+Each document type can have a unique sequence per journal where it is assigned. The data is created
+automatically when the localization module is installed, and the information required for the
+document types is included by default.
+
+.. image:: uruguay/document-types.png
+   :align: center
+   :alt: Document types for Uruguay,
+
+Sales journals
+~~~~~~~~~~~~~~
+
+To generate and confirm an electronic document that will be validated by |DGI|, the sales journal
+needs to be configured with the following:
+
+- :guilabel:`Invoicing Type`: manual invoices are for open invoices previously stamped in another
+  system, for example directly in the |DGI|. Electronic invoices will not be sent to Uruware.
```

Comment:
This is a little confusing, because it says electronic invoices will not be sent to Uruware, but if they select "electronic" they will be sent to Uruware, maybe rephrase this paragraph to explain the manual option with "manual invoices are for open invoices previously stamped in another system, for example in the DGI" and then explain the electronic option.

Reply:
Please rephrase bullet point @samueljlieber 

- :guilabel:'Invoicing Type': This option will have 'Electronic' value set by default. This is necessary to send electronic documents via web service to the Uruguayan government. The other option ('Manual') is for open invoices previously stamped in another system, for example, in the DGI.

Reply:
@samueljlieber somehow this was shown as resolved but it wasn't, can you please rephrase to:

:guilabel:'Invoicing Type': This option will have 'Electronic' value set by default. This is necessary to send electronic documents via web service to the Uruguayan government through Uruware. The other option ('Manual') is for open invoices previously stamped in another system, for example, in the DGI.

Reply:
This one also @samueljlieber can you please check? 😉